{
  "schema_version": "1.4.0",
  "id": "GHSA-pp84-v3mw-gg4w",
  "modified": "2024-10-09T19:42:44Z",
  "published": "2024-08-27T19:50:54Z",
  "aliases": [

  ],
  "summary": "Taipy 3.1.1 affected by CVEs on flask-core and pymongo",
  "details": "### Summary\nCVEs on 3.1.1\n\nFixed on patch versions: >=3.1.2 \nand on major releases: >=4.0.0\n\n### Details\n\n### CVE-2024-1681: flask-core <4.0.1 \nlatest version of taipy 3.1.1 needs <=4.0.0 \n\n### CVE-2024-5629: pymongo <4.6.3 \nlatest version of taipy 3.1.1 needs <=4.6.1 \n\n### PoC\nplease upgrade to these versions \n\n### Impact\npre-commit breaks",
  "severity": [

  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "taipy"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "4.0.0"
            }
          ]
        }
      ],
      "database_specific": {
        "last_known_affected_version_range": "<= 3.1.1"
      }
    }
  ],
  "references": [
    {
      "type": "WEB",
      "url": "https://github.com/Avaiga/taipy/security/advisories/GHSA-pp84-v3mw-gg4w"
    },
    {
      "type": "PACKAGE",
      "url": "https://github.com/Avaiga/taipy"
    }
  ],
  "database_specific": {
    "cwe_ids": [

    ],
    "severity": "HIGH",
    "github_reviewed": true,
    "github_reviewed_at": "2024-08-27T19:50:54Z",
    "nvd_published_at": null
  }
}